Java/LinkedList/MiddleLL.java

```diff
@@ -0,0 +1,72 @@
+/*
+Solution : Tortoise-Hare-Approach
+
+Unlike the above approach, we don’t have to maintain node count here and we will be able to find the middle node in a single traversal 
+so this approach is more efficient.
+
```

Comment:
Since there is no above approach here, this has no meaning. Please remove this comment.